docs/en/essentials/passing-props.md

```diff
@@ -27,6 +27,13 @@ const User = {
 const router = new VueRouter({
   routes: [
     { path: '/user/:id', component: User, props: true }
+    
+    // for routes with named views, you have to define the props option for each named view:
+    {
+      path: '/user/:id', 
+      components: { default: User, sidebar: Sidebar },
+      props: {user: true, sidebar: true }
```

Comment:
> Object mode

> When props is an object, this will be set as the component props as-is. Useful for when the props are static.

It would set `props.user` and `props.sidebar` to `true`. Right?